src/index.js

```diff
@@ -41,7 +41,9 @@ async function toggleWindowCollapse() {
 }
 
 async function loopWindows(extensionAPI) {
-  let sidebar = window.roamAlphaAPI.ui.rightSidebar.getWindows();
+  let sidebar = window.roamAlphaAPI.ui.rightSidebar.getWindows()
+                  // only remove unpinned sidebar windows
+                  .filter(w => !(w["pinned?"] || w["pinned-to-top?"]));
```

Comment:
the main fix
Since this is effectively what is happening anyways, this will work both in prod right now and after the later change